README.rst

```diff
@@ -47,7 +47,7 @@ you can run::
 
 .. note::
 
-    On OS X, if you see an error regarding the version of six that came with
+    On macOS, if you see an error regarding the version of six that came with
```

Comment:
Why change "OS X" to "macOS" here?

Reply:
That's what it's called now: 
https://www.apple.com/macos/high-sierra/
https://www.wired.com/2016/06/apple-os-x-dead-long-live-macos/

Reply:
iOS, macOS, tvOS. Makes sense. Wonder what they'll call the successor to the iPhone X…